vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
++----------+------------------------------------------------------------------------------+---------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+: {tbl-colwidths="\[17,22,15,22,25\]"}
+
+
+## QC Findings and Common Issues
+
+In this Pilot 3 study, our focus was to create ADaMs based on Pilot 1 where after ADaM generation we compared them against the analysis datasets used in Pilot 1 as a QC step. With these comparisons we listed the QC Findings with explanations as to why these findings exist. We also came across common issues throughout the ADaM generation process, which could be helpful for improvements utilising the CDISC Pilot data in the future. More details can be found in the appendix (Appendix 2 and Appendix 3).
+
+[QC Findings](https://github.com/RConsortium/submissions-pilot3-adam/wiki/QC-Findings)
+
+[Common Issues](https://github.com/RConsortium/submissions-pilot3-adam/wiki/Common-Issues)
+
+{{< pagebreak >}}
+
+# Submission of Programs
+
+## Description
+
+The sponsor has provided all programs for analysis results. They are all created on a Linux platform using R version 4.2.3.
+
+## ADaM Programs
+
+The following table contains the list of programs that generate the analysis datasets in the R Consortium R submission Pilot 3. It shows the program file name, the analysis dataset name and the label of the analysis dataset. The recommended steps to execute the analysis results using R are described in the Appendix.
+
++------------------+---------------------------+--------------------------------------+
+| **Program Name** | **Analysis Dataset Name** | **Analysis Dataset Label**           |
++==================+===========================+======================================+
+| adsl.r           | adsl.xpt                  | Subject-Level Analysis Dataset       |
++------------------+---------------------------+--------------------------------------+
+| adadas.r         | adas.xpt                  | ADAS-Cog Analysis                    |
++------------------+---------------------------+--------------------------------------+
+| adlbc.r          | adlb.xpt                  | Analysis Dataset Lab Blood Chemistry |
++------------------+---------------------------+--------------------------------------+
+| adae.r           | adae.xpt                  | Adverse Events Analysis Dataset      |
++------------------+---------------------------+--------------------------------------+
+| adtte.r          | adtte.xpt                 | AE Time to 1st Derm. Event Analysis  |
++------------------+---------------------------+--------------------------------------+
+
+## Analysis Output Programs
```

Comment:
This rendering in a bad  way for me

![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/8a27898d-20c6-4c06-ad50-69ce50a890d1)

Reply:
We may need to play around with the table widths as was done in line 229 of `adrg.qmd`. The 'Analysis Datasets' table.  We can copy this line down to the other tables to fix the widths.